protoc-gen-swagger/genswagger/template.go

```diff
@@ -175,6 +179,25 @@ func renderMessagesAsDefinition(messages messageMap, d swaggerDefinitionsObject,
 		if err := updateSwaggerDataFromComments(&schema, msgComments); err != nil {
 			panic(err)
 		}
+		opts, err := extractSchemaOptionFromMessageDescriptor(msg.DescriptorProto)
+		if opts != nil {
+			if err != nil {
```

Comment:
Can you promote error checking to immediately after the call to extractSchemaOptionFromMessageDescription?